web/themes/joinup/joinup_theme.theme

```diff
@@ -89,11 +103,27 @@ function joinup_theme_preprocess_page(&$variables) {
 
     if (in_array($rdf_type, ['collection', 'solution'])) {
       $variables['has_vertical_tabs'] = TRUE;
+      $is_form_container = TRUE;
+    }
+  }
+
+  // Check pattern for add pages.
+  $form_pattern = '.add';
```

Comment:
I checked this pattern and I think that it works ok. It doesn't depend only on length of a string. It checks if string is not shorter than '.add', and then function substr_compare checks if '.add' is at the end of a string.
But if you think that it it not good solution, I could replace it by regex.